src/select/src/Select.js

```diff
@@ -105,13 +105,14 @@ class Select extends PureComponent {
     const textSize = theme.getTextSizeForControlHeight(height)
     const borderRadius = theme.getBorderRadiusForControlHeight(height)
     const iconSize = theme.getIconSizeForSelect(height)
+    const iconMargin = height >= 36 ? 12 : 8
 
     return (
       <Box
         display="inline-flex"
         flex={1}
         position="relative"
-        width={200}
+        width="auto"
```

Comment:
This is up for debate, but if you are using a native select, this is closer to the normal behavior. I would suggest if we want a particular size we may implement that via `minWidth` instead. But I like the `width="auto"` better (because it explicitly says we want auto-sized width based on the select's option contents.